src/indexing.jl

```diff
@@ -129,8 +129,8 @@ function Base.to_index(X::NullableArray)
     Base.to_index(X.values)
 end
 
-@doc """
-`nullify!(X::NullableArray, I...)`
+"""
+   nullify!(X::NullableArray, I...)
 
 This is a convenience method to set the entry of `X` at index `I` to be null
 """ ->
```

Comment:
The arrows at the end of the doc strings should also go; they aren't part of the new syntax.